.gitignore

```diff
@@ -118,3 +118,14 @@ venv.bak/
 
 #sphinx build
 /docs/_build/
+/docs/build
+
+# Downloaded datasets for docs
+/docs/source/notebooks/aclImdb
+/docs/source/notebooks/data
+/docs/source/notebooks/*.tar.gz
+/docs/source/notebooks/spoken_digits
+/docs/source/notebooks/pretrained_models
+
+# VS Code
+.vscode/
```

Comment:
newline missing

Reply:
what is being generated in this `vscode/` directory?

Reply:
> newline missing

added a new line at the end of .gitignore.

> what is being generated in this `vscode/` directory?

currently, it just points where the doc's `conf.py` is stored relative to the workspace folder:
![image](https://user-images.githubusercontent.com/90272361/155696002-14770b37-0c9f-4a37-ab1a-3b20da0c1230.png)